src/DynamicEvaluator/Application/Checker/BrokenPhpFilesChecker/BrokenPhpFilesChecker.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace DynamicEvaluator\Application\Checker\BrokenPhpFilesChecker;
+
+use DynamicEvaluator\Application\Checker\BrokenPhpFilesChecker\Dto\ViolationDto;
+use DynamicEvaluator\Application\Checker\BrokenPhpFilesChecker\FileErrorsFetcher\FileErrorsFetcherInterface;
+
+class BrokenPhpFilesChecker
+{
+    /**
+     * @var \DynamicEvaluator\Application\Checker\BrokenPhpFilesChecker\FileErrorsFetcher\FileErrorsFetcherInterface
+     */
+    protected FileErrorsFetcherInterface $fileErrorsFetcher;
+
+    /**
+     * @param \DynamicEvaluator\Application\Checker\BrokenPhpFilesChecker\FileErrorsFetcher\FileErrorsFetcherInterface $fileErrorsFetcher
+     */
+    public function __construct(FileErrorsFetcherInterface $fileErrorsFetcher)
+    {
+        $this->fileErrorsFetcher = $fileErrorsFetcher;
+    }
+
+    /**
+     * @param array<string> $composerCommands
+     *
+     * @return array<\DynamicEvaluator\Application\Checker\BrokenPhpFilesChecker\Dto\ViolationDto>
+     */
+    public function check(array $composerCommands): array
```

Comment:
This method doesn't check anything. I would move this wrapper to fileErrorsFetcher. Doesn't make sense to keep class for it. and rename the method 'get', 'extract'. it sounds like bool return type

Reply:
This a checker facade. It return the violations. The method check should be there. The same with other checkers